src/mocks.ts

```diff
@@ -10,12 +10,28 @@ export interface Mock {
   delay?: number
 }
 
-const addDelay = (delay: number) => new Promise((res) => setTimeout(res, delay));
+export interface Scenarios {
+  'default': Mock[];
+  [scenario: string]: Mock[];
+}
+
+export const extractScenarioFromLocation = (location: Location): string => {
+  const queryString = location.search;
+  let scenario = queryString.split('?scenario=')[1];
```

Comment:
I don't think this will work if:
* The user has > 1 query param & the `scenario` param is not in the first postion.

eg: 
`test4.com?x=123&scenario=blah`

We could possibly use the `query-string` lib here, although, I'm not sure how big that lib is.

Reply:
I agree with @mrkidsan . I think it'd be better if we handled `scenario` being present anywhere in the query string. 
As to using `query-string` or some other library here, I think it's worthwhile. I don't think size matters much in our case, luckily.

Reply:
Yep that's a good shout (both of you)

Re. size - you're right @kirjai - it shouldn't matter since this should only be used with dev builds.

I'll change it so it uses `query-string` now and that `scenario` can be present anywhere in the query string. That'll get the job done nicer.